src/Command/GenerateClientCommand.php

```diff
@@ -2,86 +2,99 @@
 
 namespace App\Command;
 
-
+use RuntimeException;
 use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
-use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class GenerateClientCommand extends ContainerAwareCommand
 {
-    /**
-     * 
-     */
-    protected function configure()
+
+    public const BASE_NAMESPACE = 'Bayne\\Telegram\\Bot';
+    public const BASE_NAMESPACE_TYPES = self::BASE_NAMESPACE . '\\Type';
+
+    protected function configure(): void
     {
-        $this
-            ->setName('generate:client')
-            ->addArgument('schema', InputArgument::REQUIRED)
-        ;
+        $this->setName('generate:client');
     }
 
-
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $schemaFile = $input->getArgument('schema');
-        $schema = file_get_contents($schemaFile);
+        $buildDirSource = $this->getContainer()->getParameter('kernel.root_dir') . '/../build/source/';
+        $jsonPath = $buildDirSource . 'schema.json';
+        $schema = file_get_contents($jsonPath);
         $schema = json_decode($schema, true);
-        
-        $buildDir = $this->getContainer()->getParameter('kernel.root_dir').'/../build';
 
-        if (false === is_dir($buildDir.'/Bayne/Telegram/Bot/Object')) {
-            mkdir($buildDir.'/Bayne/Telegram/Bot/Object', 0777, true);
+        $buildDir = $this->getContainer()->getParameter('kernel.root_dir') . '/../build';
+        $baseDir = $buildDir . '/' . str_replace('\\', '/', self::BASE_NAMESPACE);
+        $baseDirTypes = $buildDir . '/' . str_replace('\\', '/', self::BASE_NAMESPACE_TYPES);
+
+        if (
+            (false === is_dir($baseDirTypes))
+            && !mkdir($concurrentDirectory = $baseDirTypes, 0777, true)
+            && !is_dir($concurrentDirectory)
+        ) {
+            throw new RuntimeException(sprintf('Directory "%s" was not created', $concurrentDirectory));
         }
 
+        $types = [
+            [
+                'AbstractInlineQueryResult',
+                ['namespace' => self::BASE_NAMESPACE_TYPES],
+            ], [
+                'AbstractInputMessageContent',
+                ['namespace' => self::BASE_NAMESPACE_TYPES],
+            ], [
+                'InputFileInterface',
+                ['namespace' => self::BASE_NAMESPACE_TYPES],
+            ], [
+                'AbstractType',
+                ['namespace' => self::BASE_NAMESPACE_TYPES],
+            ],
+        ];
+
+        $clients = [
+            [
+                'ClientInterface',
+                ['namespace' => self::BASE_NAMESPACE, 'schema' => $schema],
+            ], [
+                'TypedClientInterface',
+                ['namespace' => self::BASE_NAMESPACE, 'schema' => $schema],
+            ],
+        ];
```

Comment:
Why two different `ClientInterface`s? If you have a use for it, I'm cool with keeping it. But otherwise let's only have the TypedClientInterface